src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,52 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.Comparator;
+import java.util.List;
+
+
+public class SortCommand extends Command{
+
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Sorts address book via the selected parameter.\n"
+        + "Parameters: [name] OR [phone] OR [email] OR [address]\n"
+        + "Example: " + COMMAND_WORD + " name";
+
+    public static final String MESSAGE_SUCCESS = "List has been sorted by %1$s";
+
+    private String sortOption;
+    private Comparator<Person> sortComparator;
+//    private int[] test = new int[4];
```

Comment:
Try to avoid leaving your personal test/debugging statements in your PRs, though commented out.